src/NOTD.SC2Map/Base.SC2Data/GameData/ValidatorData.xml

```diff
@@ -551,6 +551,12 @@
         <CombineArray value="UnitisPathfinder"/>
         <CombineArray value="TargetIsEnemyorNeutral"/>
     </CValidatorCombine>
+    <CValidatorPlayerCompareResource id="PlayerHas5000XP">
+        <WhichPlayer Value="Target"/>
+        <Compare value="GE"/>
+        <Resource value="Custom"/>
+        <Value value="5000"/>
+    </CValidatorPlayerCompareResource>
```

Comment:
This is the validator that checks player XP.
"Custom" resource *is* NOTD "XP", "GE" is ">=" (greater than or equal to)